src/helpers/index.ts

```diff
@@ -55,7 +48,7 @@ export const getBase255Number = (color: string, alpha = false): number => {
 
 
 //---Calculate a decimal 0-1 value from CMYK value
-export const getCMYKNumber = (color: string): number => Math.min(PCENT.test(color) ? +(color.replace(PCENT, '$1')) / 100 : +color, 1);
+export const getCMYKNumber = (color: string): number => Math.min(PCENT.test(color) ? percent(color) / 100 : +color, 1);
```

Comment:
The `pcent` utility takes care of the first part of the code I think. Maybe it could be shortened in:

```
export const getCMYKNumber = (color: string): number => Math.min(percent(color) / 100, 1);
``` 

Test it on your side to see if it covers all the edge cases.

Reply:
I tested it and using the code you mentioned breaks the tests, the thing is that the "percent" function just translates a percentage string into a number, it doesn't divide it by 100

Reply:
Yes I got it. It is just that it checks for `PCENT.test(color)` and executes a function that checks again for the same. I would rather separate the logic of `+(color.replace(PCENT, '$1'))` into a separated utility function and use it in:

* helpers `percent`
* helpers `getBase255Number`
* helpers `getCMYKNumber`
* utils `normalizeAlpha`

Reply:
fixed as requested